Library/Formula/fuzzy-choose.rb

```diff
@@ -0,0 +1,17 @@
+class FuzzyChoose < Formula
+  desc "Hybrid string fuzzy matcher that uses both std{in,out} and a native GUI"
```

Comment:
@bfontaine Actually I worded it the way I do so it would fit in a "Show HN" post title, which has an 80-char limit (including the "Show HN: " part). So currently it already is 71 characters. `echo -n "Hybrid string fuzzy matcher that uses both std{in,out} and a native GUI" | wc -c` => `71`